src/VisualStudio/LiveShare/Impl/Client/Tagger/ModeAwareSyntacticClassificationViewTaggerProvider.cs

```diff
@@ -17,7 +18,7 @@ namespace Microsoft.VisualStudio.LanguageServices.LiveShare.Client.Tagger
     [Export(typeof(IViewTaggerProvider))]
     [PartCreationPolicy(CreationPolicy.Shared)]
     [TagType(typeof(IClassificationTag))]
-    [ContentType(StringConstants.CSharpLspContentTypeName)]
+    [ContentType(ContentTypeNames.CSharpLspContentTypeName)]
     [TextViewRole(PredefinedTextViewRoles.Document)]
     internal sealed class CSharpModeAwareSyntacticClassificationViewTaggerProvider : ModeAwareSyntacticClassificationViewTaggerProvider
```

Comment:
Why is there both a view tagger provider and a regular tagger provider? I would have assumed this was because of a syntax versus semantic classification split, but it seems both are syntax related?

(This question is orthogonal to the PR really but somewhat confused of what I'm looking at here.)

Reply:
Orthogonal, but generally this does remote syntax classifications first based on the current view, then for the entire document (for the remote classification exp).  This is because the regular tagger is too chatty, it makes a lot of requests for very small ranges in succession, which causes large delays over the network.  TS implemented something similar for their oop classifications